modules/datadog/templates/cluster_role.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
Since these are not template files, would manifests/files be a better name for the folder?

Reply:
👍